### PR TITLE
Feat : Added Auth.

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Login and Signup features added.

### CODE REVIEW

File: src/Pages/Authentication/Signup/signup.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            onChange={e => setUser({ ...user, firstName : e.target.value })} required />
+
+<div class="input-labels">Last name</div>
+        <input class="input-field"  value={user.lastName}
+            onChange={e => setUser({ ...user, lastName : e.target.value })} required />
+            
+        
+        <div class="input-labels">Email Address</div>
+        <input class="input-field" type="email" placeholder="example@gmail.com" value={user.email}
+            onChange={e => setUser({ ...user, email: e.target.value })} required />
```
Comment (vishalg8454) [inline src/Pages/Authentication/Signup/signup.js:53]: you can extract this to a function.

**Review by vishalg8454 — APPROVED** [review]
Clean code. Good to merge. Can look at the one suggestion I gave.

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+import { Signin } from "./Pages/Authentication/Login/login";
+import {  Routes , Route } from "react-router-dom"
+import { Signup } from "./Pages/Authentication/Signup/signup";
 
 function App() {
   return (
     <div className="App">
+      <Routes>
+        <Route path = "/login" element = {<Signin />} />
+        <Route path = "/signup" element = {<Signup />} />
```
Comment (shivanigangadharan) [inline src/App.js]: Can ommit `/` here as it's not reqd in react router v6

File: src/Services/signin-service.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const signinService = async ({ email, password }) => {
+  try {
+    const { data } = await axios.post("/api/auth/login", {
+      email: email,
+      password: password,
+    });
+    console.log(data.foundUser)
```
Comment (shivanigangadharan) [inline src/Services/signin-service.js]: can remove unwanted consoles

File: src/Services/signup-service.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    const {
+      data
+    } = await axios.post("/api/auth/signup", {
+      email: email,
+      password: password,
+      firstName : firstName , 
+      lastName : lastName
+    });
+    console.log(data)
```
Comment (shivanigangadharan) [inline src/Services/signup-service.js]: here too

**Review by shivanigangadharan — APPROVED** [review]
Suggested minor changes, rest of the code looks well structured. 
Good logics used. 👍🏼

**Pull request merged**